modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/alert-info/configuration.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Just a suggestion: 

alert-info seems be something from bootstrap, it could be named `welcome` or something related?